PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/Invisicam.rbxmx

```diff
@@ -165,6 +164,21 @@ end
 
 -- Update. Called every frame after the camera movement step
 function Invisicam:Update()
+	-- Make sure we still have a Torso
+	if not Torso or not Torso.Parent then
+		local humanoid = Character:FindFirstChild("Humanoid")
+		if humanoid and humanoid.Torso then
+			Torso = humanoid.Torso
+		else
+			-- Not set up with Humanoid? Try and see if there's one in the Character at all:
+			Torso = Character:FindFirstChild("Torso")
```

Comment:
This probably should be HumanoidRootPart. The original code *was* using Torso though, so I made it identical.